.github/workflows/perf-testing-setup.yml

```diff
@@ -2,7 +2,6 @@ name: Throughput testing workflow
 
 on:
   workflow_dispatch:
-  push:
```

Comment:
Removed throughput tests for each push. `workflow_dispatch` allows having a manual run on any branch.